pipeline/coordinator.go

```diff
@@ -39,6 +39,11 @@ const (
 	StrategyFallbackExternal Strategy = "fallback_external"
 )
 
+const (
+	// Only mp4s of maxMP4OutDuration will have MP4s generated for each rendition
+	maxMP4OutDuration = 2 * time.Minute
```

Comment:
We may consider making this part of the TranscodeJobArgs instead of a fixed value into catalyst-api? Not necessarily in this pr
LGTM, in any case